schedule/yast/autoyast/autoyast_sap.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: create_hdd_sap
+description: >
+  SAP autoYaST installation and publish qcow for SAP migration standalone
+  cases.
+vars:
+  AUTOYAST: autoyast_sle15/create_hdd/create_hdd_sap.xml.ep
+  AUTOYAST_PREPARE_PROFILE: '1'
+  NW: 'nfs://1c119.qa.suse.de/srv/nfs/sap/NW75_CLUSTER'
+  INSTANCE_ID: '00'
+  INSTANCE_SID: 'QAD'
+  INSTANCE_TYPE: 'ASCS'
+  QEMU_VIRTIO_RNG: '0'
+schedule:
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/wicked
+  - autoyast/repos
+  - autoyast/clone
+  - autoyast/logs
+  - '{{handle_reboot}}'
+  - '{{test_sles4sap}}'
+  - console/system_prepare
+  - console/hostname
+  - console/force_scheduled_tasks
+  - '{{handle_deregistration}}'
+  - shutdown/grub_set_bootargs
+  - shutdown/cleanup_before_shutdown
+  - shutdown/shutdown
+
+conditional_schedule:
+  handle_reboot:
+    ARCH:
+      x86_64:
+        - autoyast/autoyast_reboot
+        - installation/grub_test
+        - installation/first_boot
+  handle_deregistration:
```

Comment:
yes, mostly we need to de-register system, but for the case "sap_migration_offline_scc_sles4sap15sp4" on PowerVM, we need to keep system registered, otherwise there is a issue happened like this https://openqa.suse.de/tests/10001310

Reply:
In fact, it is better to setup independent schedule file for PowerVM or Z/VM, they are so different :)

Reply:
yes, it is good idea if there are many differences, but the same are more in these cases, and I think it is better to use only one file before we don't make things complicated enough, and it is flexible if we can control registration, what do you think?

Reply:
yes, I agree with you, but I think the yaml file is simple enough to this ticket.